### DISCUSSION TITLE
Moving to the RuboCop organization?

### DISCUSSION THREAD

**Discussion by bbatsov** [rubocop/rubocop#9517] [Maintenance]
I guess at some point we'll need to decide if we want to switch the GitHub org we're using from `rubocop-hq/rubocop` (https://github.com/rubocop-hq), to `rubocop/rubocop` (https://github.com/rubocop), now that we control both of them. 

I don't have a strong preference for this, although I can see the (some?) benefit of having a more "canonically" named organization. As GitHub redirects are going to happen automatically that's not going to cause issues for the users, but it's going to create some work for us, as it's going to break temporarily things like CI. 

@rubocop-hq/rubocop-core Feel free to share your general thoughts on the subject and to vote with 👍 or 👎 for moving to the `rubocop` org.

**Comment by marcandre** [comment]
👉

**Comment by koic** [comment]
I feel that users were able to keep up with the change that made rubocop repo managed by rubocop-hq org about three years ago. And user impact will be less when the rename is early.
https://github.com/rubocop-hq/rubocop/pull/5935

I think rubocop/rubocop is the more natural and best org/repo 👍

**Comment by marcandre** [comment]
Meta: should we lock this conversation, or delete it, as there doesn't seem to be a way to "resolve"/close it...
